twig/js/ledere_leder.html.twig

```diff
@@ -54,24 +54,42 @@
 				<thead>
 					<tr>
 						<th width="*%">Overnattingssted</th>
-						{% for dato in netter %}
-							<th width="40">
-								{{ dato|dato('D') }}
-								<small>{{ dato|dato('d.m') }}</small>
-							</th>
-						{% endfor %}
+						{% if netterAntall > 0 %}
```

Comment:
Hvordan blir netterAntall tilgjengeliggjort for twig-template når den rendres fra php @slogan-15 ?

Reply:
netterAntall kommer fra JS. Se `handleLederCreate()` @mariusmandal

Reply:
Ser det funker fra både php og js, så da tenker jeg det er ok @slogan-15 😊 

Men, jeg fikk denne bug'en når jeg la til en ny leder. Kan du fikse det? Siste overnatting mangler navn på dag (søndag 07.03). Når jeg refresher siden, så virker det.
<img width="1414" alt="Skjermbilde 2021-03-24 kl  11 22 24" src="https://user-images.githubusercontent.com/4747102/112294773-7838c680-8c93-11eb-9d9e-3d58827835c4.png">

Reply:
Commit 775d1f7 skal fikse det tror jeg 🙂

Reply:
Testet ok :)